src/main/resources/bidder-config/pubmatic.yaml

```diff
@@ -1,6 +1,17 @@
 adapters:
   pubmatic:
     endpoint: https://hbopenbid.pubmatic.com/translator?source=prebid-server
+    aliases:
+      groupm:
+        enabled: false
+        endpoint:
```

Comment:
Please, specify endpoint url if it differs from `adapters.pubmatic.endpoint`.
Otherwise, PBS won't startup with `URL supplied is not valid` error.

Or just omit the property if the same with `pubmatic` one.